build-system/pr-check/checks.js

```diff
@@ -81,6 +81,10 @@ async function prBuildWorkflow() {
     timedExecOrDie('amp ava');
   }
 
+  if (buildTargetsInclude(Targets.BUILD_SYSTEM)) {
+    timedExecOrDie('amp check-build-system');
```

Comment:
Curious about the overall running time of this task during CI. (I sent you https://github.com/ampproject/amphtml/pull/34557, which should shave off several seconds by skipping the unnecessary NPM checks.)

Reply:
After patching your PR it runs in 8 seconds locally and about 1 minute in CI.

https://app.circleci.com/pipelines/github/ampproject/amphtml/9989/workflows/c86536b8-50c8-4789-83e2-47b94cb531dc/jobs/155206
https://app.circleci.com/pipelines/github/ampproject/amphtml/9988/workflows/5801ebc2-034f-4de2-8eb7-2523df0d77b7/jobs/155156

Given that it is much faster than lint, what would you think about running this check first during `prBuildWorkflow`?

Reply:
> After patching your PR it runs in 8 seconds locally and about 1 minute in CI.

These are excellent results! I assume the 8 seconds is because the npm packages were previously installed (which is exactly as desired).

> Given that it is much faster than lint, what would you think about running this check first during `prBuildWorkflow`?

I have another idea to speed up linting for the majority of CI builds: If a PR modifies only a few lint files and doesn't touch any lint rules or upgrade any packages, we can simply run `amp lint --local_changes`, which will finish in seconds. (There's no need to run prettier on the entire repo all the time.) Will send a separate PR for this.

Reply:
Yes, that is with npm packages installed. Running `amp lint --local_changes` sounds like a great idea.